pkg/kubernetes/pod/pod.go

```diff
@@ -116,10 +120,12 @@ func (spc *SimplePodContext) Delete(ctx context.Context, name, namespace string)
 // NewPodExecutor creates a new SimplePodExecutor.
 func NewPodExecutor(client client.Client, config *rest.Config, name, namespace string) (*SimplePodExecutor, error) {
 	return &SimplePodExecutor{
-		name:      name,
-		namespace: namespace,
-		client:    client,
-		config:    config,
+		name:         name,
+		namespace:    namespace,
+		client:       client,
+		config:       config,
+		IntervalWait: 2 * time.Second,
+		TimeoutWait:  30 * time.Second,
```

Comment:
30 seconds seem too much, lets default this to 15 for the time being
```suggestion
		IntervalWait: 3 * time.Second,
		TimeoutWait:  15 * time.Second,
```